apps/io-services-cms-webapp/src/historicizer/request-historicization-handler.ts

```diff
@@ -44,7 +47,6 @@ export const handleQueueItem = (context: Context, queueItem: Json) =>
     queueItem,
     parseIncomingMessage(Queue.RequestHistoricizationItem),
     TE.fromEither,
-    TE.mapLeft((_) => new Error("Error while parsing incoming message")), // TODO: map as _permanent_ error
```

Comment:
**question:** Why is this line no longer useful?

Reply:
same as the previous comment!

Reply:
👌  replied in this [comment](https://github.com/pagopa/io-services-cms/pull/990#discussion_r1743350955)